.gitignore

```diff
@@ -9,3 +15,20 @@
 # CI files
 /dist
 profile.cov
+
+# Test binary, build with `go test -c`
+*.test
+
+# Output of the go coverage tool, specifically when used with LiteIDE
+*.out
+
+# Project-local glide cache, RE: https://github.com/Masterminds/glide/issues/736
+.glide/
+
```

Comment:
We use Go mod, this is outdated.

```suggestion
```

Reply:
Ok removed that. Was from my default one ;)